cmd/go/of_functions_framework/BUILD.bazel

```diff
@@ -8,9 +8,9 @@ licenses(["notice"])
 buildpack(
     name = "of_functions_framework",
     srcs = [
-        "converter/get_package/main.go",
         "converter/without-framework/go.mod",
         "converter/without-framework/main.go",
+        "//cmd/go/of_functions_framework/converter/get_package:main.go",
```

Comment:
`//cmd/go/of_functions_framework/converter/get_package:main.go"` can be changed to 
`converter/get_package:main.go"` to be consistent with others?

Reply:
What's the reason to change `converter/get_package/main.go` to `//cmd/go/of_functions_framework/converter/get_package:main.go`?

Reply:
> What's the reason to change `converter/get_package/main.go` to `//cmd/go/of_functions_framework/converter/get_package:main.go`?

here I actually just follow the upstream: https://github.com/GoogleCloudPlatform/buildpacks/blob/main/cmd/go/functions_framework/BUILD.bazel#L13, but I think `converter/get_package:main.go` should be fine as well. I will test it locally.

Reply:
ok, let's keep the same as upstream to avoid future conflict